lib/findpath.mjs

```diff
@@ -0,0 +1,44 @@
+import { dirname } from 'node:path';
+import { env, platform } from 'node:process';
+import { join, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+/**
+ * Get the platform dependant path of the NW.js binary.
+ * 
+ * @param {'nwjs' | 'chromedriver'} exe Path to NW.js or Chromedriver executable.
+ * @return {string}
+ */
+export default function findpath(exe = 'nwjs') {
+  const __dirname = dirname(fileURLToPath(import.meta.url));
+  const nwDir = resolve(__dirname, '..', 'nwjs');
+  /**
+   * File path to executable.
+   * 
+   * @type {string}
+   */
+  let binPath = '';
+
+  /**
+    * Host operating system
+    * 
+    * @type {NodeJS.Platform | 'osx' | 'win'}
+    */
+  let hostOs = env.npm_config_nwjs_platform || env.NWJS_PLATFORM || platform;
```

Comment:
process.platform can only return one of these values:

* 'aix'
* 'android'
* 'darwin'
* 'freebsd'
* 'linux'
* 'openbsd'
* 'sunprocess'
* 'win32'

So it will never `=== 'osx'`

Reply:
Thanks for catching this!

Reply:
Wasn't talking about the JSDoc, look at the code.